sql/item_latents.sql

```diff
@@ -686,18 +682,22 @@ INSERT INTO `item_latents` VALUES(16238, 369, 1, 13, 3);
 INSERT INTO `item_latents` VALUES(14954, 5, 35, 14, 0);
 INSERT INTO `item_latents` VALUES(14954, 168, -5, 14, 0);
 INSERT INTO `item_latents` VALUES(14954, 71, 1, 14, 0);
-INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 2);
+
+-- -------------------------------------------------------
+-- Sirius Axe
+-- -------------------------------------------------------
 INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 3);
 INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 4);
 INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 5);
+INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 6);
 
 -- -------------------------------------------------------
 -- Company Sword
 -- -------------------------------------------------------
-INSERT INTO `item_latents` VALUES(17662, 366, 2, 15, 2); -- DMG 46~54, increases by 2 for every person in party (excl. wearer)
```

Comment:
nevermind, wikis don't state same thing, ffxiclopedia says "over 2" and bg doesn't - its right as you have it pretty sure..